deps/edu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Installing required packages
+apt --yes install devscripts equivs
+
+# Building debian-edu packages
+cd /vagrant/deps/edudebian-meta-1.0
```

Comment:
Is this version number needed? When you make a new version of the package you don't make a new directory, do you?
What does this `deps` mean? I think it should be `packages` or `deb`.

Reply:
`deps` can be renamed to something like `edu-packages`.

Reply:
>Is this version number needed?

No, it will work without it as well. Just give us a warning.
```
dpkg-source: warning: source directory 'edudebian-meta' is not <sourcepackage>-<upstreamversion> 'edudebian-meta-1.0'
```
But as we are usiing these packages for our own purposes we can do that.